src/util/output/error.ts

```diff
@@ -1,4 +1,15 @@
 import chalk from 'chalk';
+import * as configFiles from '../config/files'
+import metrics from '../metrics';
+import { GA_TRACKING_ID } from '../constants'
+
+const config: any = configFiles.getConfigFilePath();
+const shouldCollectMetrics = (
```

Comment:
Can we share the logic for checking if metrics are enabled? It's currently in two files.